CMakeLists.txt

```diff
@@ -374,6 +374,52 @@ else ()
   set(broker_includes ${CMAKE_CURRENT_SOURCE_DIR}/auxil/broker/include ${CMAKE_CURRENT_BINARY_DIR}/auxil/broker/include)
 endif ()
 
+if ( NOT DISABLE_SPICY )
+    if ( NOT Spicy_ROOT )
+        add_subdirectory(auxil/spicy)
+
+        # Set variables used by the spicy-plugin build since we are building Spicy
+        # as part of Zeek so spicy-plugin cannot use `spicy-config` at configure
+        # time to set these.
+        set(SPICY_CONFIG "<builtin>")
+        set(HAVE_SPICY "YES")
+        set(SPICY_HAVE_TOOLCHAIN "YES")
+        set(SPICY_INCLUDE_DIRS_RUNTIME
+            ${CMAKE_SOURCE_DIR}/auxil/spicy/spicy/hilti/runtime/include
+            ${CMAKE_SOURCE_DIR}/auxil/spicy/spicy/spicy/runtime/include
+            ${CMAKE_BINARY_DIR}/auxil/spicy/spicy/include)
+        set(SPICY_INCLUDE_DIRS_TOOLCHAIN
```

Comment:
On macos with Homebrew one currently needs to unlink both the `zeek` and `spicy` packages if they are installed as otherwise the spicy-plugin build picks up incorrect include paths. Let's check whether we can have our own include paths earlier in the path so that issue does not come up anymore.

Reply:
Will add a patch addressing this.